AppCenter/AppCenter/Internals/History/Device/MSDeviceTracker.m

```diff
@@ -73,6 +74,13 @@ - (void)setWrapperSdk:(MSWrapperSdk *)wrapperSdk {
   }
 }
 
+- (void)setCountryCode:(NSString *)countryCode{
```

Comment:
Nitpick : space before `{`

Reply:
@Quan-Jin Please run `./clang-format-changed-files.sh` from root directory to format your code before submitting the PR. The script will take care of any styling issues.